### PR TITLE
Remove slashes from DCR

### PR DESCRIPTION
**Pull request by abeddow91**
## What does this change?
This PR moves the slash design change from behind its 0% opt in test. It also removes the slash css code and the showSlash prop which is no longer necessary. 

As per the designs, we now have a line break where components formally had a slash. The Card component previously had a showSlash prop but it was not being used. As such, we have not implemented a hideLineBreak prop on this component as it also not be used - the card component's kicker is always followed by a line break as designs currently stand.

## Screenshots

Screenshot 2023-01-30 at 14 22 16
Screenshot 2023-01-30 at 14 22 08
Screenshot 2023-01-30 at 14 27 14

### CODE REVIEW

**Comment by bryophyta** [comment]
From CODE deployment:

image
image
image
image
image

File: dotcom-rendering/src/web/components/Card/Card.tsx [hunk 1]
```diff
@@ -58,8 +58,6 @@ export type Props = {
 	showMainVideo?: boolean;
 	kickerText?: string;
 	showPulsingDot?: boolean;
-	/** Sometimes kickers and headlines are separated by a slash */
-	showSlash?: boolean;
 	/** Sometimes kickers and headlines are separated by a linebreak */
 	hideLineBreak?: boolean;
```
Comment (bryophyta) [inline dotcom-rendering/src/web/components/Card/Card.tsx:62]: I wonder whether we should remove this as a prop from the Card component? As things stand, most uses of `Card` are wrapped in `FrontCard`, which doesn't have the prop on it. And I'm not aware of any cases where a card shouldn't have a linebreak between kicker and title for its main headline (as opposed to for its sublinks). What do you think?

Comment (abeddow91) [reply dotcom-rendering/src/web/components/Card/Card.tsx:62]: Brilliant! Yep, I'll remove it, the fewer props the better.

**Review by ioannakok — APPROVED** [review]
Great work!

**Pull request merged**